src/examples/component-definition/json/example-component-with-config.json

```diff
@@ -44,7 +44,7 @@
                     },
                     {
                         "uuid": "22dbff65-9729-449f-9dfc-4e5fee0906de",
-                        "source": "../../../content/fedramp.gov/json/FedRAMP_HIGH-baseline_profile.json",
+                        "source": "https://raw.githubusercontent.com/GSA/fedramp-automation/master/baselines/rev4/json/FedRAMP_rev4_HIGH-baseline_profile.json",
```

Comment:
@ohsh6o Is this the best (most stable and authoritative) version of this file to reference?

Reply:
At this time, yes. There is a pending release tag for the profiles and other data, but it is not ready at this time. I can PR again once that changes to mark it to something more stable. Is that ok, @david-waltermire-nist?